### ISSUE TITLE
New Term: Impaired heel-walking

### ISSUE THREAD

**Issue by CorneliusDreyling** [obophenotype/human-phenotype-ontology#10311]
**Preferred term label:**
Impaired heel-walking ability

**Synonyms**
no synonyms

**Definition (free text, please give PubMed ID)**
Difficulty or reduced ability to walk on heels. Heel-walking can be tested as a part of the neurological examination. Foot dorsalextention weakness leads to difficulties in walking on heels. PMID: 29291292

**Parent term (use hpo.jax.org/app)**
Gait disturbance HP:0001288

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0009-0004-9942-3886

**Comment by pnrobinson** [comment]
@CorneliusDreyling 
Adding new term: 
- ID: HP:6000490
- label: Impaired heel-walking ability
- parents: [Gait disturbance (HP:0001288)](https://hpo.jax.org/app/browse/term/HP:0001288)
- pmid: PMID:29291292
- def: Difficulty or reduced ability to walk on heels. Heel-walking can be tested as a part of the neurological examination. Foot dorsal extention weakness leads to difficulties in walking on heels.
- comment: 
- orcid: orcid:0009-0004-9942-3886
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10311
- synonyms:
